<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>second_window</class>
 <widget class="QMainWindow" name="second_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>second_window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="data_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>741</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>741</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="link_lay"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>450</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="main_buttons_lay"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>741</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="label_lay"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="show_extra_settings_lay"/>
    </widget>
    <widget class="QWidget" name="settings_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>401</width>
       <height>91</height>
      </rect>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="max_clients_lay"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>401</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="camera_index_lay"/>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="role_buttons_widget" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>661</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>661</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="role_buttons_lay">
      <item>
       <widget class="QPushButton" name="create_button">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>200</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>20</pointsize>
          <italic>true</italic>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Create the conference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="join_button">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>200</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>20</pointsize>
          <italic>true</italic>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Join the conference</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>10</y>
      <width>187</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="return_lay">
     <item>
      <widget class="QPushButton" name="return_button">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>return to a start menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1257</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHH">
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>11</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="title">
     <string>HH</string>
    </property>
   </widget>
   <addaction name="menuHH"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
